app/assets/stylesheets/sephora_style_guide/app/pagination.scss

```diff
@@ -0,0 +1,65 @@
+#pagination {
+  section {
+    float: left;
+    width: 50%;
+  }
+  
+  .pagination {
+    display: inline-block;
+    text-align: center;
+    
+    .page {
+      background: $white;
+      border: 1px solid $grey-mid;
+      border-right: 0;
+      display: inline-block;
+      font-size: 13px;
+      text-align: center;
+      text-decoration: none;
+      padding: 5px;
+      height: 23px;
+      min-width: 23px;
+      width: auto;
+      
+      &:last-child {
+        border-right: 1px solid $grey-mid;
+      }
+      
+      a, em {
```

Comment:
Usually would not prefer element selectors but in this particular case should be fine. Could I add a new line after `a,`